website/pages/docs/migration/stitches.md

```diff
@@ -292,16 +292,19 @@ const styles = css({
 })
 ```
 
-Notice that in Panda, you don't need to use the `$` prefix to access the tokens. If you really want to use the `$` prefix, update the name of the token.
+Notice that in Panda, you don't need to use the `$` prefix to access the tokens. If you really want to use the `$` prefix, you can format how the tokens, classes and css vars will be generated.
+
+```ts
+const dasherize = (token) => token.toString()
+  .replace(/[^a-zA-Z0-9]+/g, '-')
+  .replace(/^-+|-+$/g, '')
 
-```diff
 export default defineConfig({
-  theme: {
-    colors: {
--      gray100: { value: 'hsl(206,22%,99%)' },
-+      $gray100: { value: 'hsl(206,22%,99%)' },
-    },
-  }
+  // Stitches preset
+  separator: '-',
```

Comment:
If we already have `.join('-')` in formatTokenName, is this still neccessary?
cc @astahmer

Reply:
the separator sits between the class prefix and the final class, the `.join('-')` is meant to join the token path